sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -194,7 +194,9 @@ public partial class RequestContext
         public RequestContext() { }
         public System.Threading.CancellationToken CancellationToken { get { throw null; } set { } }
         public Azure.ErrorOptions ErrorOptions { get { throw null; } set { } }
+        public bool HasCustomClassifier { get { throw null; } }
```

Comment:
I'm not wild about adding this, but we need it to be public to access it in LLC-generated code, or else I think we'd have to pass in a factory lambda to create a new instance.

Reply:
Why generated code needs this? I was looking at usage in this PR and it seems like one method uses it that we could make reusable.